src/ivc/cyclefold/mod.rs

```diff
@@ -20,11 +20,11 @@ pub const RATE: usize = T - 1;
 pub const R_F: usize = 10;
 pub const R_P: usize = 10;
 
-/// Safety: because 32 != 0
-pub const DEFAULT_LIMB_WIDTH: NonZeroUsize = unsafe { NonZeroUsize::new_unchecked(32) };
+/// Safety: because 64 != 0
```

Comment:
what is the reason to increase both LIMB_WIDTH and COUNT_LIMIT here?

Reply:
This simply reduced the number of rows, the search for optimal values is left to the benchmark stage